Ginger/Ginger/UserControlsLib/ucGridView/ucGrid.xaml.cs

```diff
@@ -1895,10 +1896,23 @@ void IDragDrop.StartDrag(DragInfo Info)
                 }
 
                 Info.DragSource = this;
```

Comment:
don't we supposed to add similar handling to ucListView? for covering multi select and drag of List View items, like in add Variabels page in which we can drag Variabels to the list

Reply:
Done !